setup.py

```diff
@@ -15,10 +15,9 @@
     #url='',
     #license='',
     packages=[
-        'rna', 'rna.migrations', 'rna.management', 'rna.management.commands'],
+        'rna', 'rna.management', 'rna.management.commands'],
     install_requires=[
-        'South',
-        'Django>=1.4.9',
+        'Django>=1.8,<1.9',
         'djangorestframework>=2.3.7',
```

Comment:
Should probably update this to `djangorestframework>=3.3.2` to match requirements/api.txt